docs/signer.md

```diff
@@ -0,0 +1,354 @@
+# Kritis Signer
+
+Kritis Signer is a command-line tool that creates attestations. These attestations can be used for Kritis and Binary Authorization deployment enforcement.
+
+The tool can create attestations directly or conditionally, checking policy conformance based on Vulnerability scanning results from Google Cloud Container Analysis.
+
+It can be run either locally or as part of a continuous integration (CI) pipeline.
+
+Currently, the tool supports vulnerability-based policy check, 
```

Comment:
Please see if this paragraph can be removed based on my addition of the paragraph above.

Reply:
Done.